worker/create_batches_from_filters_worker.go

```diff
@@ -171,8 +186,10 @@ func (b *CreateBatchesFromFiltersWorker) preprocessPages(job *model.Job, stageSt
 
 func (b *CreateBatchesFromFiltersWorker) processPages(c <-chan DBPage, writeToCSVCH chan<- *[]User, job *model.Job, wg *sync.WaitGroup, wgCSV *sync.WaitGroup, stageStatus *StageStatus) {
 	for page := range c {
+		PrintMemUsage()
 		users := b.getPageFromDBWithFilters(job, page)
 		b.Logger.Info("got users from db", zap.Int("usersInBatch", len(*users)))
+		b.Workers.Statsd.Incr("processPages", []string{GetPushDBTableName(job.App.Name, job.Service)}, 1)
```

Comment:
Statsd metrics should be in snake case, so "process_pages", "write_user_page_into_csv", etc